client/src/pages/profile/index.tsx

```diff
@@ -0,0 +1,494 @@
+                      max={new Date(Date.now()).toISOString().split('T')[0]}
+                    />
+                  </div>
+                ) : (
+                  <div className="input-field">
+                    <span>{`${parseCamelCase('dateJoined')}:`}</span>
+                    <Input
+                      value={new Date(user.dateJoined).toLocaleDateString(
+                        'en-US',
+                        { timeZone: 'UTC' },
+                      )}
+                      transparent
+                      readOnly={!isEditMode}
+                    />
+                  </div>
+                )}
+              </Container>
+            </Grid.Column>
+            <Grid.Column textAlign="center" width={3}>
+              <HeaderTag as="h2">My Interests</HeaderTag>
+              {!isEditMode ? (
+                user.interests.map((interest, index) => (
+                  <div
+                    className={`field-label ${convertToClassName(interest)}`}
+                    key={index}
+                  >
+                    {convertToCapitalized(interest)}
+                  </div>
+                ))
+              ) : (
+                <Container style={{ paddingLeft: '20%' }}>
+                  {allInterests.map((interest, index) => (
+                    <div key={index} style={{ textAlign: 'left' }}>
+                      <Checkbox
+                        value={interest}
+                        label={convertToCapitalized(interest)}
+                        checked={user.interests.includes(interest)}
+                        onChange={(e, data) => editInterests(`${data.value!}`)}
+                      />
+                    </div>
+                  ))}
+                </Container>
+              )}
+            </Grid.Column>
+            <Grid.Column textAlign="center" width={3}>
+              <HeaderTag as="h2">My Roles</HeaderTag>
+              {!isEditMode || currentUser.role !== 'ADMIN' ? (
+                user.currentTeams.map((team, index) => (
+                  <div
+                    className={`field-label ${convertToClassName(team)}`}
+                    key={index}
+                  >
+                    {convertToCapitalized(team)}
+                  </div>
+                ))
+              ) : (
+                <Container style={{ paddingLeft: '20%' }}>
+                  {allTeams.map((team, index) => (
+                    <div key={index} style={{ textAlign: 'left' }}>
+                      <Checkbox
+                        value={team}
+                        label={convertToCapitalized(team)}
+                        checked={user.currentTeams.includes(team)}
+                        onChange={(e, data) => editTeams(`${data.value!}`)}
+                      />
+                    </div>
+                  ))}
+                </Container>
+              )}
+            </Grid.Column>
+          </Grid.Row>
+          <Divider />
+          <Grid.Row columns={2}>
```

Comment:
I think it would look better if this section was centered so that it aligns better with the content above the divider?